float64 des_p_x
float64 des_p_y
float64 des_p_z

float64 des_v_x
float64 des_v_y
float64 des_v_z

float64 des_a_x
float64 des_a_y
float64 des_a_z

float64 yaw
